src/testInput/java/components/README.md

```
@@ -1,5 +1,6 @@
-Most of the classes in this directory come from the Oracle Java Tutorial
-[Using Swing Components: Examples](https://docs.oracle.com/javase/tutorial/uiswing/examples/components/index.html)
+Most of the classes in this directory come from the Oracle Java Tutorial [Using
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)